core/nodejsActionBase/src/service.js

```diff
@@ -160,6 +160,17 @@ function NodeActionService(config) {
     };
 
     function doInit(message) {
+        if (message.env && typeof message.env == 'object') {
+            Object.keys(message.env).forEach(k => {
```

Comment:
Could have used [Object.entries][1] but looks like thats only from Nodejs 7+. So used approach is more compatabile

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Reply:
if/when we drop node 6, can switch it.